Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -673,6 +676,13 @@ void UpdateSelectionMode()
 			}
 		}
 
+		void UpdateSpinnerColor()
+		{
+			var color = Element.RefreshControlColor;
+
+			if (_tableViewController != null)
+				_tableViewController.UpdateRefreshControlColor(color.ToUIColor(UIColor.Black));
```

Comment:
Instead of `UIColor.Black` you need to find someway to find the initial color of the `RefreshControl`, then on this method when `color == Color.Default` you call `UpdateRefreshControlColor` with that color that you capture initially as the default color of the spinner. If you use Color.Default it's transparent so the spinner disappears, you need to capture the native initial color.

Reply:
I think I see your point. According to this link, it is nil as a default: https://developer.apple.com/documentation/uikit/uirefreshcontrol/1624847-tintcolor so would that make sense then? Just tested it and set the tint color to null (when the property is Color.Default) makes the spinner black.

I should double check how it is on Android.